plugins/services/images/local.go

```diff
@@ -214,15 +211,3 @@ func (l *local) Delete(ctx context.Context, req *imagesapi.DeleteImageRequest, _
 
 	return &ptypes.Empty{}, nil
 }
-
-func (l *local) emitSchema1DeprecationWarning(ctx context.Context, image *images.Image) {
-	if image == nil {
-		return
-	}
-	dgst, ok := image.Labels[images.ConvertedDockerSchema1LabelKey]
-	if !ok {
-		return
-	}
-	log.G(ctx).WithField("name", image.Name).WithField("schema1digest", dgst).Warn("conversion from schema 1 images is deprecated")
-	l.warnings.Emit(ctx, deprecation.PullSchema1Image)
```

Comment:
Can you also remove `deprecation.PullSchema1Image`?

Reply:
The PR no longer removes Schema 1